drivers/mtd/Kconfig

```diff
@@ -345,6 +345,16 @@ config NULLMTD_ERASESTATE
 
 endif # NULLMTD
 
+config RPMSGMTD
```

Comment:
MTD_RPMSG ?

Reply:
Done

Reply:
Both are good for me
MTD_RPMSG: like MTD_PARTITION, implement an abstract layer upper on an exists mtd device, MTD_RPMSG implement an abstract layer upper on a remote mtd device;
RPMSGMTD: like RAMMTD, an independent mtd device implementation.

Reply:
@xiaoxiang781216 @anchao Should i change it back?

Reply:
Yes, BTW, since the file name is rpmsgmtd.c which is more consistent with RPMSGMTD.

Reply:
Done